std-bits/base/src/main/java/org/enso/base/enso_cloud/logging/LogJobsQueue.java

```diff
@@ -1,16 +1,15 @@
-package org.enso.base.enso_cloud.audit;
+package org.enso.base.enso_cloud.logging;
 
 import java.util.ArrayList;
 import java.util.Deque;
 import java.util.LinkedList;
 import java.util.List;
 
-class LogJobsQueue {
-
-  private final Deque<AuditLogApiAccess.LogJob> queue = new LinkedList<>();
+public final class LogJobsQueue {
```

Comment:
Why this class is exported? Make package private.

Reply:
Done in 039b27949d88dee15ef785f348fff2c4fe329043